opm/material/common/Tabulated1DFunction.hpp

```diff
@@ -129,16 +129,17 @@ class Tabulated1DFunction
                          bool sortInputs = true)
     {
         assert(x.size() == y.size());
-        assert(x.size() > 1);
 
         resizeArrays_(x.size());
-        std::copy(x.begin(), x.end(), xValues_.begin());
-        std::copy(y.begin(), y.end(), yValues_.begin());
-
-        if (sortInputs)
-            sortInput_();
-        else if (xValues_[0] > xValues_[numSamples() - 1])
-            reverseSamplingPoints_();
+        if (x.size() > 0) {
+            std::copy(x.begin(), x.end(), xValues_.begin());
+            std::copy(y.begin(), y.end(), yValues_.begin());
+
+            if (sortInputs)
+                sortInput_();
+            else if (xValues_[0] > xValues_[numSamples() - 1])
```

Comment:
What's the value of `numSamples()` here?  If it's less than `1` this will be very bad.

Reply:
i haven't messed with that, but we do not hit this code (it's surrounded by a newly added if()).

Reply:
(value will be 0 which is why i had to if it out). the point here is that these sanity checks are not in place when default-constructing the function, but bit now when i'm recreating the empty function in deserialization.